roles/strimzi_kafka/defaults/main.yaml

```diff
@@ -15,3 +15,8 @@ strimzi_kafka_create_prometheus_instance: true
 
 strimzi_kafka_install_grafana: true
 strimzi_kafka_grafana_prometheus_url: http://prometheus-operated.kafka:9090
+
+strimzi_kafka_add_feature_gates: true
+strimzi_kafka_feature_gates:
+  - "+UseKRaft"
```

Comment:
```suggestion
- "+UseKRaft"
```

Reply:
It seems we generally indent lists, e.g. [here](https://github.com/SeldonIO/ansible-k8s-collection/blob/master/playbooks/seldon_core.yaml#L4) and [here](https://github.com/SeldonIO/ansible-k8s-collection/blob/master/playbooks/seldon_core.yaml#L22)